radis/lbl/broadening.py

```diff
@@ -697,26 +696,48 @@ def _whiting_jit(w_centered, wl, wv):
     # ... fasten up the calculation by 25% (ex: test on 20 cm-1, ~6000 lines:
     # ... 20.5.s > 16.5s) on the total eq_spectrum calculation
     # ... w_wv is typically a (10.001, 1997) array
-    # from time import time
-    a = time()
+    #   from time import time
+    #   print("w_centered : ", w_centered)
+    #   print("w_centered shape : ", np.shape(w_centered))
+    #   print("wl shape : ", np.shape(wl))
+    #   print("wv shape : ", np.shape(wv))
+    # a = time()
```

Comment:
@anandxkumar can you entirely remove the commented prints ? (you can get them back from the Git history if needs be!)